pandas/core/generic.py

```diff
@@ -487,24 +487,30 @@ def _expand_axes(self, key):
 
         return new_axes
 
-    _shared_docs['set_axis'] = """Assign desired index to given axis
+    _shared_docs['set_axis'] = """
+        Assign desired index to given axis.
+
+        Indexes for columns or rows can be changed by assigning a list-like or Index.
 
         Parameters
         ----------
-        labels: list-like or Index
-            The values for the new index
-        axis : int or string, default 0
+        labels : list-like, Index
+            The values for the new index.
+
+        axis : int, default 0
```

Comment:
axis : {0 or 'index', 1 or 'columns'}, default 0

Reply:
And modify the description to say that the string forms are also acepted.

Reply:
Thanks, so axis : {0 or 'index', 1 or 'columns'}, default 0 is enough or should I mentioned `int, str` too?